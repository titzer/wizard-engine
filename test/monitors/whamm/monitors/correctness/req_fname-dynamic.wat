(module
  (import "wizeng" "puts" (func $puts (param i32 i32)))
  (memory (export "engine:data") 1 1)
  (data (i32.const 20) "\n")

  (func (export "wasm:opcode:call (fname)") (param i32 i32))
)
