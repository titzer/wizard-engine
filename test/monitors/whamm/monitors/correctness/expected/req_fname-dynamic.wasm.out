monitor:0x2: error inserting Whamm! probe whamm probe: have not supported using fname dynamically yet.
